### DISCUSSION TITLE
Nginx proxy [Jun 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2434] [Discord Threads]
> 🙋‍♂️ **NoobieDev**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### Can you try to access `https://domainname/nocodb/dashboard`  
#  
> 🙋‍♂️ **NoobieDev**  
#### Thanks for the response <@682854952657616902>  .  
#  
> 🙋‍♂️ **NoobieDev**  
#### When i do that is says , im guessing the extra slash is the issue  
![image0](https://cdn.discordapp.com/attachments/984719048208162838/984728454702460948/unknown.png)  
#  
> 👨‍💻 **pranavxc**  
#### okkk  
#  
> 👨‍💻 **pranavxc**  
#### `proxy_pass http://localhost:8086/;` => `proxy_pass http://localhost:8086;`  
#  
> 🙋‍♂️ **NoobieDev**  
####   
![image0](https://cdn.discordapp.com/attachments/984719048208162838/984729812318953482/unknown.png)  
#  
> 🙋‍♂️ **NoobieDev**  
#### the first slash is gone , but still same error . tried /nocodb/dashboard/ as well  
#  
> 👨‍💻 **pranavxc**  
#### nginx configuration looks good 🤔  
#  
> 🙋‍♂️ **NoobieDev**  
#### the redirection to /dashboard does not happen now.  
#  
> 👨‍💻 **pranavxc**  
#### if you are directly accessing dashboard path then it won't redirect `https://domainname/nocodb/dashboard`  
#  
> 🙋‍♂️ **NoobieDev**  
#### okay , I had the same issue when i tried deploying it as node app using pm2.  
#  
> 👨‍💻 **pranavxc**  
#### could you able to access directly without ngnix ?  
#  
> 🙋‍♂️ **NoobieDev**  
#### i was able to access with location /  on a fresh server.  
#  
> 🙋‍♂️ **NoobieDev**  
#### i have my api running on  ``` loaction /api```  
#  
> 👨‍💻 **pranavxc**  
#### can you check the docker logs as well  
#  
> 👨‍💻 **pranavxc**  
#### `docker logs -f nocodb`  
#  
> 🙋‍♂️ **NoobieDev**  
####   
![image0](https://cdn.discordapp.com/attachments/984719048208162838/984746929722388530/unknown.png)  
#  
> 🙋‍♂️ **NoobieDev**  
#### Mybad this is the wrong one  
#  
> 🙋‍♂️ **NoobieDev**  
####   
![image0](https://cdn.discordapp.com/attachments/984719048208162838/984747721418235904/unknown.png)  
#  
> 🙋‍♂️ **NoobieDev**  
#### FYI the last 2 Get is me trying using Curl  
#  
> 👨‍💻 **pranavxc**  
#### Can you try the following configuration   
####    
#### ```location /nocodb/ {   
####                    
####                 proxy_set_header Host $host;   
####             proxy_set_header X-Forwarded-Host $host:$server_port;   
####             proxy_set_header X-Forwarded-Proto https;   
####             proxy_set_header X-Real-IP $remote_addr;   
####             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;   
####             proxy_pass http://localhost:8086/;   
####             chunked_transfer_encoding off;   
####             proxy_buffering off;   
####             proxy_cache off; }   
#### }   
#### ```  
#  
> 👨‍💻 **pranavxc**  
#### or     
####    
#### ```location ~/nocodb(.*)$ {   
####                    
####                 proxy_set_header Host $host;   
####             proxy_set_header X-Forwarded-Host $host:$server_port;   
####             proxy_set_header X-Forwarded-Proto https;   
####             proxy_set_header X-Real-IP $remote_addr;   
####             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;   
####             proxy_pass http://localhost:8086$1;   
####             chunked_transfer_encoding off;   
####             proxy_buffering off;   
####             proxy_cache off; }   
#### }   
#### ```  
#  
> 🙋‍♂️ **NoobieDev**  
#### Didnt try this yet , but as a workaround i created another subdomain and put the nocodb in another server block. That is working fine .  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/984719048208162838)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
